hail/python/hailtop/hailctl/dev/benchmark/run/matrix_table_benchmarks.py

```diff
@@ -36,10 +36,18 @@ def matrix_table_rows_force_count():
     ht._force_count()
 
 
+def _range_matrix_table_p100():
+    mt = hl.utils.range_matrix_table(n_rows=1_000_000, n_cols=10, n_partitions=100)
+    return mt.annotate_entries(x=mt.col_idx + mt.row_idx)
+
+
+RANGE_MATRIX_TABLE_P100 = _range_matrix_table_p100()
```

Comment:
pretty sure you cant do this -- it will force initialize hail when the matrix_table_benchmarks file is imported, and that breaks how benchmark works.

Reply:
would have been fine to leave as an abstracted def, but fine to revert too.